lib/action.js

```diff
@@ -1,11 +1,11 @@
 const core = require('@actions/core');
 
-// Matches [ch49555] and ch49555/branch-name
-const clubhouseRegex = /(\[ch\d+\])|(ch\d+\/)/;
+// Matches [sc-49555] and sc-49555/branch-name
```

Comment:
It looks like it would match `sc49555` rather than `sc-49555`?

Also, do we want to match either `chXXXXX` _or_ `scXXXXX` for some time to be able to transition?

Reply:
I like that idea!

Reply:
I love that idea! Shortcut's own docs list `sc-`